### ISSUE TITLE
Unable to place cursor close to beginning of line

### ISSUE THREAD

**Issue by ruedigerkupper** [ubports/telegram-app#192]
Device: hammerhead
Version: OTA-4

While typing a message I am unable to place the cursor at the first 3 to 4 characters in any line. I have no problem at all placing it at a later point, but not close to the start.

This does not occur in other apps (like in the browser app I am typing right now). And it is *no hardware issue* with my device, since it also appears in landscape mode where the left edge of the entry field is located at the middle of the screen. It must be a software issue.

**Comment by advocatux** [comment]
You can't place the cursor tapping but you can place it using the keyboard in _joystick mode_ or whatever is that mode called (press&hold space bar, the keyboard goes blank, move the cursor like a joystick)

**Comment by ruedigerkupper** [comment]
@advocatux
But that means you confirm this bug? On which device and build?

**Comment by advocatux** [comment]
Hmm interesting, today I can place the cursor tapping at the beginning of the line both in krillin & hammerhead, both running xenial devel up to date, but yesterday I couldn't do it.

The intriguing part is that hammerhead has been updated today but krillin has not.

Anyway _the tapping to put the cursor at the beginning_ works pretty unreliable and erratic. Afaik it has been always that way, even in vivid.

More than a bug this is a little annoyance to me.

**Comment by ruedigerkupper** [comment]
Yes, it probably was there from the beginning. I never paid much attention, but the more often you type messages, the more annoying it gets.
Does anyone have an idea what causes this problem? This is probably some standard editable field (sorry, I'm not so much in mobile programming). It strikes me as very strange that it has problems with cursor positioning? If so, this will probably occur elsewhere?
